build-system/app.js

```diff
@@ -1331,4 +1331,21 @@ function decryptDocumentKey(encryptedDocumentKey) {
   return parsedJson.key;
 }
 
+// serve local vendor config JSON files
+app.use('(/dist)?/rtv/*/v0/analytics-vendors/:vendor.json', (req, res) => {
```

Comment:
Curious when will the request contain `/dist/`?

Reply:
when `isLocalDev` is true, `calculateScriptBaseUrl()` will add `/dist` to the base URL

Reply:
In that case should, is the `/dist` always there?